src/sentry/similarity.py

```diff
@@ -143,6 +143,50 @@ def delete(self, scope, items, timestamp=None):
             arguments,
         )
 
+    def export(self, scope, items, timestamp=None):
+        if timestamp is None:
+            timestamp = int(time.time())
+
+        arguments = [
+            'EXPORT',
+            timestamp,
+            len(self.bands),
+            self.interval,
+            self.retention,
+            scope,
+        ]
+
+        for idx, key in items:
+            arguments.extend([idx, key])
```

Comment:
Yeah, this is entirely for readability since I didn't actually write documentation or type annotations for any of this.